bin/cron/update_pledge_map

```diff
@@ -26,34 +26,60 @@ CENSUS_FORM_KINDS = "'Census2017','HocCensus2017'".freeze
 file = Tempfile.new(['data', '.csv'])
 begin
   CSV.open(file.path, 'w') do |csv|
+    DASHBOARD_DB[:census_submissions].
+      select_append(:census_submissions__id___census_submissions_id).
```

Comment:
Does this need to be `select_append` as opposed to `select`? `select_append` will generate `SELECT *,census_submissions.id`. Not sure you need the *?

Reply:
Yes, this can be select instead of select_append. I just needed to list each column explicitly.